packages/training/python/scripts/local-finetune/test_adapter.py

```
@@ -0,0 +1,91 @@
+
+def get_device():
+    if torch.cuda.is_available(): return "cuda"
+    if torch.backends.mps.is_available(): return "mps"
+    return "cpu"
+
+def main():
+    device = get_device()
+    print(f"Running on: {device}")
+    
+    if device != "cuda":
+        model.to(device)
```

Ruff (0.14.8)

Undefined name `model`

(F821)